samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -46,7 +46,7 @@
    * Defines the comma separated list of job status servers of the form
    * "Host1:Port1,Host2:Port2".
    */
-  private static final String CONFIG_JOB_STATUS_SERVERS = "job.status.servers";
+  private static final String CONFIG_JOB_STATUS_SERVERS = "jobs.status.servers";
```

Comment:
The recent for renaming is for consistency. 

All other config keys that are in this Config class have prefix jobs(and this has prefix job). 
* How this collides with JobsResourceConfig? 
* Why this is confusing? (Reason for extracting this out separately is to group all LocalStore Monitor config together. May be we could consider renaming all the config within this class to keep it different from JobsResourceConfig).